products/sle15/profiles/pci-dss.profile

```diff
@@ -69,13 +70,14 @@ selections:
     - audit_rules_usergroup_modification_shadow
     - audit_rules_sysadmin_actions
     - display_login_attempts
+    - ensure_gpgcheck_never_disabled
     - file_groupowner_grub2_cfg
     - file_owner_grub2_cfg
     - gid_passwd_group_same
     - no_empty_passwords
     - rpm_verify_permissions
     - service_auditd_enabled
     - service_pcscd_enabled
+    - set_password_hashing_algorithm_libuserconf
```

Comment:
this rule needs to be updated with `sle15` `prodtype`. https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_libuserconf/rule.yml#L3